client/app/templates/Components/Alerts/_validation-summary.html.twig

```diff
@@ -10,7 +10,7 @@
                 {% endfor %}
             {% endif %}
             {% for formErrorMessage in formErrorMessages %}
-                <li><a href="#form-group-{{ formErrorMessage.elementId }}">{{ formErrorMessage.message | nl2br}}</a></li>
+                <li><a href="#{{ formErrorMessage.elementId }}" aria-invalid="true" aria-describedby="{{ formErrorMessage.elementId }}-error">{{ formErrorMessage.message | nl2br}}</a></li>
```

Comment:
I think this should be on the element linked to, not on the link. "The aria-describedby attribute lists the ids of the elements that describe the object." In this case, the object is the form element, and the element that describes it is the error message.